frontend/public/components/resource-quota.jsx

```diff
@@ -82,66 +83,68 @@ const ResourceUsageRow = ({quota, resourceType}) => {
   </div>;
 };
 
-const QuotaScopes = ({quota}) => {
-  const scopes = _.get(quota, ['spec', 'scopes']);
-  if (_.isEmpty(scopes)) {
-    return <dd>-</dd>;
-  }
+export const QuotaGaugeCharts = ({quota, resourceTypes, rowHeight, wrapperClass}) => {
+  const resourceTypesSet = new Set(resourceTypes);
+  const rowHeightClass = `co-resource-quota-chart-row--${rowHeight}`;
+  return <div className={classNames('co-resource-quota-chart-row', {[rowHeightClass]: rowHeight})} >
+    {(resourceTypesSet.has('requests.cpu') || resourceTypesSet.has('cpu')) &&
+    <div className="co-resource-quota-gauge-chart">
+      <Gauge title="CPU Request" thresholds={gaugeChartThresholds} wrapperClass={wrapperClass}
+        percent={getResourceUsage(quota, resourceTypesSet.has('requests.cpu') ? 'requests.cpu' : 'cpu').percent} />
+    </div>}
+    {resourceTypesSet.has('limits.cpu') &&
+    <div className="co-resource-quota-gauge-chart">
+      <Gauge title="CPU Limit" thresholds={gaugeChartThresholds} wrapperClass={wrapperClass}
+        percent={getResourceUsage(quota, 'limits.cpu').percent} />
+    </div>}
+    {(resourceTypesSet.has('requests.memory') || resourceTypesSet.has('memory')) &&
+    <div className="co-resource-quota-gauge-chart">
+      <Gauge title="Memory Request" thresholds={gaugeChartThresholds} wrapperClass={wrapperClass}
+        percent={getResourceUsage(quota, resourceTypesSet.has('requests.memory') ? 'requests.memory' : 'memory').percent} />
+    </div>}
+    {resourceTypesSet.has('limits.memory') &&
+    <div className="co-resource-quota-gauge-chart">
+      <Gauge title="Memory Limit" thresholds={gaugeChartThresholds} wrapperClass={wrapperClass}
+        percent={getResourceUsage(quota, 'limits.memory').percent} />
+    </div>}
+  </div>;
+};
+
+export const QuotaScopes = ({scopes, compact}) => {
   return scopes.map(scope => {
     const scopeObj = _.get(quotaScopes, scope);
     return scopeObj ?
-      <dd key={scope}>
-        <div className="co-resource-quota-scope__label">{scopeObj.label}</div>
-        <div className="co-resource-quota-scope__description">{scopeObj.description}</div>
+      <dd key={scope} className={classNames({'quota-dashboard-scopes--compact':compact})}>
+        <div className={classNames('co-resource-quota-scope__label', {'co-resource-quota-scope__label--concise':compact})}>{scopeObj.label}</div>
+        <div className={classNames('co-resource-quota-scope__description', {'co-resource-quota-scope__description--concise':compact})}>{scopeObj.description}</div>
       </dd>
       : <dd key={scope} className="co-resource-quota-scope__label">{scope}</dd>;
   });
 };
 
-const showCharts = resourceTypes => {
+export const showCharts = resourceTypes => {
```

Comment:
This needs a better name if we export. Maybe somthing like

```suggestion
export const hasComputeResources = resourceTypes => {
```

Reply:
renamed